src/content/en/2020/mobile-web.md

```diff
@@ -22,3 +22,278 @@ queries: 12_Mobile_Web
+* [HTTP archive](https://almanac.httparchive.org/en/2020/methodology#dataset)
+* [Lighthouse](https://almanac.httparchive.org/en/2020/methodology#lighthouse)
+
+Please visit the links above to learn more about the methodology and caveats with each data source.
+It is worth noting that HTTP Archive and Lighthouse data is limited to the data identified from websites’ home pages only, and not site-wide. 
+
+In addition to the above, we also used a non-public Chrome data source in the section on Page loads in Chrome. For more information on this, read about [Chrome’s data collection API](https://chromium.googlesource.com/chromium/src/+/master/services/metrics/ukm_api.md).
+
+While this data is only collected from a subset of (opted in) Chrome users, it does not suffer from being limited to homepages. It is pseudonymous and consists of histograms and events. 
+
+NOTE: Reporting is enabled if the user has enabled a feature that syncs browser windows, unless they have disabled the "Make searches and browsing better / Sends URLs of pages you visit to Google" setting.
+
+## Mobile web & Desktop traffic trends
+How much are users visiting websites on mobile web and desktop? Are there any patterns in the traffic that websites receive from mobile vs. desktop?  In order to examine these questions and what it means for websites, we looked at data from a couple of lenses.
+
+A [report published](https://www.perficient.com/insights/research-hub/mobile-vs-desktop-usage-study) on perficient.com shows mobile vs. desktop traffic trends over several years, using [similarweb](https://www.similarweb.com/) as a data source.
+While the majority of visits -- **58%** of site visits -- were from mobile devices, mobile devices made up only 42% of total time spent online. Moreover, the average time spent per visit is roughly twice as much on desktop compared to mobile (11.52 minutes on desktop vs. 5.95 minutes on mobile). 
+
+### Page loads in Chrome (Chrome data source)
+Note that this section references stats that have been made available specifically for this chapter from non-public Chrome data source, [see details here](#datasources).
+We use this data to assess page loads on Android and Windows -- as a proxy for mobile and desktop respectively. 
+
+NOTE: we may refer to the data in this section as mobile for Android and desktop for Windows.
+
+#### Page loads across origins ranked by popularity
+
+We looked at traffic to origins by popularity -- how often are users visiting certain origins, and what does that tell us about the global distribution across the web.
+
+Rick Byers [tweeted](https://twitter.com/RickByers/status/1195342331588706306) this distribution a year ago, we looked at the latest data.
+The chart shows us the overall distribution across origins by their popularity, captured by their contribution to % page loads in Chrome.
+
+TODO: Insert chart: Page loads across origins ranked by popularity
```

Comment:
Jinja-style comments will preserve the TODOs in the markdown but prevent them from rendering in the HTML.
```suggestion
{# TODO: Insert chart: Page loads across origins ranked by popularity #}
```

Reply:
updated